openedx/core/djangoapps/schedules/management/commands/send_course_update.py

```diff
@@ -0,0 +1,14 @@
+from openedx.core.djangoapps.schedules.management.commands import SendEmailBaseCommand
+from openedx.core.djangoapps.schedules.resolvers import CourseUpdateResolver
+
+
+class Command(SendEmailBaseCommand):
+    resolver_class = CourseUpdateResolver
+
+    def __init__(self, *args, **kwargs):
+        super(Command, self).__init__(*args, **kwargs)
+        self.log_prefix = 'Upgrade Reminder'
+
+    def send_emails(self, resolver, *args, **options):
+        for day_offset in xrange(-7, -77, -7):
```

Comment:
Hrmm... Wait, this actually seems kind of hard. How do we know how many tasks to schedule?

Reply:
This seems like another reason to make the initial split be per-course, so that then you can have the course, and then trigger tasks for each week.

Reply:
As it is right now, this will generate 240 tasks: 24 hours * 10 weeks.

Reply:
Yes.  I see what you're saying.  We can be smarter about how many weeks a course actually has and create tasks accordingly.